px-vis-worker.js

```diff
@@ -495,11 +495,15 @@ function calcDataSingleQuadtree(rawData, k, visData, renderedData) {
         renderedValue = value;
       }
   //for each point calc its coords
-  return {
-    "coord": visData.radial ? calcPixelCoordForRadial(value[x], value[y], axis, visData) : [ visData.xScale(renderedValue[x]), visData.yScale[axis](renderedValue[y]) ],
+  var returnData = {
+    "coord": [],
     "name": k,
     "value": value
   }
+  if(visData.radial || (renderedValue[y] != null && renderedValue[y] != undefined) ) {
```

Comment:
do you need undefined check here as well, since it is already handled before for coords calculation.

Reply:
There was issue with passing undefined value to yScale function to get the coord. Hence adding a check here.